src/VolleyManagement.Backend/VolleyManagement.Services/GameService.cs

```diff
@@ -837,31 +837,31 @@ private Game GetNextLoserGame(Game finishedGame, List<Game> games)
             return nextGame;
         }
 
-        private int GetNextGameNumber(Game finishedGame, List<Game> games)
+        private int GetNextGameNumber(Game finishedGame, ICollection<Game> games)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Consider using more general type 'System.Collections.Generic.IEnumerable<VolleyManagement.Domain.GamesAggregate.Game>' instead of 'System.Collections.Generic.ICollection<VolleyManagement.Domain.GamesAggregate.Game>'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS3242)

Reply:
?